types/node/v12/globals.d.ts

```diff
@@ -1187,6 +1187,10 @@ declare namespace NodeJS {
         constructor(id: string, parent?: Module);
     }
 
+    interface Dict<T> {
```

Comment:
Is there a reason to add this instead of just using `Record<string, T | undefined>` or `Partial<Record<string, T>>` below?

Reply:
It’s used in the **v13** and **v14** types:
<https://github.com/DefinitelyTyped/DefinitelyTyped/blob/0017f70d8d2100bf40d49316b654d61273fadcd7/types/node/globals.d.ts#L599-L605><https://github.com/DefinitelyTyped/DefinitelyTyped/blob/0017f70d8d2100bf40d49316b654d61273fadcd7/types/node/v13/globals.d.ts#L1130-L1136>

Reply:
I also don't believe those are truly necessary, but I supposed they could all be removed in a later PR.